cmake/multi_image.cmake

```diff
@@ -85,6 +93,12 @@ function(add_child_image_from_source)
     message(FATAL_ERROR "Missing parameter, required: NAME SOURCE_DIR")
   endif()
 
+  # Pass information that the partition manager is enabled to Kconfig.
+  add_overlay_config(
```

Comment:
Curious: Why isn't this possible with `SHARED_MULTI_IMAGE_VARIABLES`

Reply:
Kconfig would not have access to the option if you pass the kconfig defines to a child image through SHARED_MULTI_IMAGE_VARIABLES. It would only be available in CMake, but not passed on to Kconfig.